app/controllers/api/top_fixed_posts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::TopFixedPostsController < Api::ApplicationController
+  def index
+    posts = current_site.top_fixed_posts.includes(post: [:categories]).published.ordered.map(&:post)
```

Comment:
It seems that we need `Post` object, not `FixedPost`.
So let create named scope for `current_site.posts`

`current_site.posts.published.ordered.fixed`

BTW, I confuse fixed_post with fixed_page :dizzy: 
I want to give another name but I have no idea :sweat: